### DISCUSSION TITLE
How to make searching with `/` case insensitive

### DISCUSSION THREAD

**Discussion by mykoza** [71/dance#275] [Q&A] [answered]
Hi,

I swear, I saw similar question before, but cannot find it again.

Is there a way to make search command `/` case insensitive?
In kakoune, one can use inline modifier `(?i)`, but from what I've found, JS regex doesn't support it.

**Comment by 71** [comment, marked as answer]
Good point, I thought I had implemented it but I hadn't. 6d305ef65e08268d0d2915cad07419b8d6a41cb5 adds support for `(?i)` in Dance commands.

**Reply by mykoza** [reply]
Thanks for quick answer.
Will there be a way to make a keybinding with it?

**Reply by meepleek** [reply]
I'm also interested in that as I would like to make the default a case insensitive search.

**Reply by 71** [reply]
Good point, there's no way to make search case-insensitive by default. That's definitely something I'd like to make possible, so I'll think about a solution.
